website/i18n/en.json

```diff
@@ -353,6 +359,12 @@
       "version-0.20/version-0.20-example-eatapple": {
         "title": "A Healthy Clause"
       },
+      "version-0.20/version-0.20-faq-developers": {
```

Comment:
this file shouldn't be changed by hand, you can revert. This will be generated by docusaurus.

Reply:
It looked to me like this was building on netlify - wouldn't you not commit the website directory and let netlify build it?